test/NuGet.Core.Tests/NuGet.Commands.Test/MSBuildRestoreUtilityTests.cs

```diff
@@ -181,10 +247,10 @@ public void MSBuildRestoreUtility_GivenDifferentProjectPathCasingsVerifyResult()
         [PlatformFact(Platform.Linux)]
         public void MSBuildRestoreUtility_GivenDifferentProjectPathCasingsVerifyPackageReferencesAreCorrect()
         {
-            using (var workingDir = TestDirectory.Create())
+            using (var workingDir = TestDirectory.Create()) 
```

Comment:
remove trailing whitespace (your editor doesn't support [`.editorconfig`'s `trim_trailing_whitespace`](https://github.com/NuGet/NuGet.Client/blob/ad08bc710b3c4e142040f4efc860c003502aee3e/.editorconfig#L13) :(

Reply:
Fixed. It's ccode...I use VS and code interchangeably....VS is problematic when you want to make many changes across files.